### PR TITLE
Basics working for Cordova iOS 16.4

### PR DESCRIPTION
**Pull request by SailingSteve**
Disabled pay-to-promote in Cordova, to prevent stripe access in iOS app Updated some Cordova components
Un-copied a bunch of styles into `CampaignDetailsStyles.jsx`, to figure out which one was effecting the markup in Cordova Un-copied 5 or 6 state choosers, to find the one that was causing trouble in Cordova.

### CODE REVIEW

File: src/js/common/components/Campaign/CampaignCardList.jsx [hunk 1]
```diff
@@ -170,7 +170,7 @@ class CampaignCardList extends Component {
           )}
         </ListWrapper>
         {/*
-        <Suspense fallback={<></>}>
+        <Sus pense fallback={<></>}>
```
Comment (DaleMcGrew) [inline src/js/common/components/Campaign/CampaignCardList.jsx]: Typo

File: src/js/components/Navigation/SettingsSectionFooter.jsx [hunk 2]
```diff
@@ -90,7 +91,7 @@ class SettingsSectionFooter extends Component {
           <Link to="/more/terms"><TermsAndPrivacyText>Terms</TermsAndPrivacyText></Link>
         </OneRow>
         <OneRow centered={centered}>
-          {(isWebApp() && !inPrivateLabelMode) && (
+          {(isWebApp() && !inPrivateLabelMode && webAppConfig.ENABLE_PAY_TO_PROMOTE) && (
```
Comment (DaleMcGrew) [inline src/js/components/Navigation/SettingsSectionFooter.jsx]: This code below isn't related to PAY_TO_PROMOTE and doesn't need to be turned off.

File: src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx [hunk 3]
```diff
@@ -673,7 +648,7 @@ class CampaignNewsItemDetailsPage extends Component {
                             </BlockedIndicator>
                           </IndicatorButtonWrapper>
                         )}
-                        {(!isSupportersCountMinimumExceeded && !inDraftMode && !inPrivateLabelMode) && (
+                        {(!isSupportersCountMinimumExceeded && !inDraftMode && !inPrivateLabelMode && webAppConfig.ENABLE_PAY_TO_PROMOTE) && (
```
Comment (DaleMcGrew) [inline src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx]: This is not related to PAY_TO_PROMOTE and doesn't need to be turned off.

File: src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx [hunk 4]
```diff
@@ -561,7 +536,7 @@ class CampaignNewsItemDetailsPage extends Component {
                         </BlockedIndicator>
                       </IndicatorButtonWrapper>
                     )}
-                    {(!isSupportersCountMinimumExceeded && !inDraftMode && !inPrivateLabelMode) && (
+                    {(!isSupportersCountMinimumExceeded && !inDraftMode && !inPrivateLabelMode && webAppConfig.ENABLE_PAY_TO_PROMOTE) && (
```
Comment (DaleMcGrew) [inline src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx]: This is not related to PAY_TO_PROMOTE and doesn't need to be turned off.

File: src/js/common/pages/Campaign/CampaignDetailsPage.jsx [hunk 5]
```diff
@@ -396,7 +397,7 @@ class CampaignDetailsPage extends Component {
                         </BlockedIndicator>
                       </IndicatorButtonWrapper>
                     )}
-                    {(!isSupportersCountMinimumExceeded && !inPrivateLabelMode) && (
+                    {(!isSupportersCountMinimumExceeded && !inPrivateLabelMode && webAppConfig.ENABLE_PAY_TO_PROMOTE) && (
```
Comment (DaleMcGrew) [inline src/js/common/pages/Campaign/CampaignDetailsPage.jsx]: This is not related to PAY_TO_PROMOTE and doesn't need to be turned off.

File: src/js/common/pages/Campaign/CampaignDetailsPage.jsx [hunk 6]
```diff
@@ -516,7 +517,7 @@ class CampaignDetailsPage extends Component {
                             </BlockedIndicator>
                           </IndicatorButtonWrapper>
                         )}
-                        {(!isSupportersCountMinimumExceeded && !inPrivateLabelMode) && (
+                        {(!isSupportersCountMinimumExceeded && webAppConfig.ENABLE_PAY_TO_PROMOTE) && (
```
Comment (DaleMcGrew) [inline src/js/common/pages/Campaign/CampaignDetailsPage.jsx]: This is not related to PAY_TO_PROMOTE and doesn't need to be turned off.

**Comment by DaleMcGrew** [comment]
Please add ENABLE_PAY_TO_PROMOTE to WebApp/src/js/config-template.js

File: src/js/common/components/Campaign/CampaignCardList.jsx (hunk 1 above)
Comment (SailingSteve) [reply src/js/common/components/Campaign/CampaignCardList.jsx]: This was not a typo, added a note.

File: src/js/components/Navigation/SettingsSectionFooter.jsx (hunk 2 above)
Comment (SailingSteve) [reply src/js/components/Navigation/SettingsSectionFooter.jsx]: Ok!

File: src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx (hunk 3 above)
Comment (SailingSteve) [reply src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx]: Ok!

Comment (SailingSteve) [reply src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx]: Ok!

File: src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx (hunk 4 above)
Comment (SailingSteve) [reply src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx]: Ok!

File: src/js/common/components/Campaign/CampaignCardList.jsx (hunk 1 above)
Comment (SailingSteve) [reply src/js/common/components/Campaign/CampaignCardList.jsx:173]: Reverted in my latest PR, with a note

File: src/js/components/Filter/StateDropDownCore.jsx [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+      dialogLabel,
+    } = this.props;
+
+    let stateCodeTemp;
+    const stateNameList = Object.values(stateCodeMap);
+
+    return (
+      <FormControl variant="outlined" className={classes.formControl}>
+        <InputLabel htmlFor="outlined-age-native-simple" />
+        <Select
```
[the remark this answers is not in the document]
Comment (SailingSteve) [reply src/js/components/Filter/StateDropDownCore.jsx:33]: I tried to add an accessible name in this PR, and the addition broke it.  I didn't put a ton of effort into it.

**Comment by SailingSteve** [comment]
@DaleMcGrew 
Didn't see this until after my follow on PR.  All resolved except that accessible label issue, which was in old code, and which a quick try broke the form.

File: src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx (hunk 3 above)
Comment (SailingSteve) [reply src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx]: Line 651

File: src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx (hunk 4 above)
Comment (SailingSteve) [reply src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx]: L 539

**Comment by SailingSteve** [comment]
Merged my 2 PRs (one of which you have not seen). Somehow lost some changes (like config-template.js change) in this PR.  Gets messy working on two source branches at once in Cordova.

**Comment by DaleMcGrew** [comment]
👍
